app/src/main/java/com/github/libretube/ui/extensions/Activity.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.libretube.ui.extensions
+
+import android.app.Activity
+import androidx.core.view.WindowCompat
+import androidx.core.view.WindowInsetsCompat
+import androidx.core.view.WindowInsetsControllerCompat
+
+fun Activity.hideSystemBars() {
+    WindowCompat.getInsetsController(window, window.decorView).apply {
+        systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
+        hide(WindowInsetsCompat.Type.systemBars() or WindowInsetsCompat.Type.navigationBars())
+    }
+}
+
+fun Activity.showSystemBars() {
+    WindowCompat.getInsetsController(window, window.decorView).apply {
+        systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
```

Comment:
```suggestion
        systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_TOUCH
```

Reply:
Also, these two methods could be easily combined into one, e.g. `toggleSystemBars(show: Boolean)`.

Reply:
No. `WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_TOUCH` conflict with touch to show player controller. i.e now we have to tap twice to show player controller